File: Assets/VirtualCity/ProtoDefine/Java/system/ReqUpdateOtherDataMessage.java
package com.kingston.jforgame.server.game.system.message;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import com.kingston.jforgame.server.game.Modules;
import com.kingston.jforgame.server.game.system.SystemDataPool;
import com.kingston.jforgame.socket.annotation.MessageMeta;
import com.kingston.jforgame.socket.message.Message;

@MessageMeta(module = Modules.BASE,cmd = SystemDataPool.REQ_OTHERDATA)
public class ReqUpdateOtherDataMessage extends Message {
    @Protobuf(order = 1)
    private String fieldName;
    @Protobuf(order = 2)
    private String value;

    public String getFieldName() {
        return fieldName;
    }

    public void setFieldName(String fieldName) {
        this.fieldName = fieldName;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }
}

File: Assets/VirtualCity/ProtoDefine/ReqUpdateOtherDataMessage.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]


public class ReqUpdateOtherDataMessage {
[ProtoMember(1)]
    public string fieldName;
[ProtoMember(2)]
    public string value;

    public string getFieldName() {
        return fieldName;
    }

    public void setFieldName(string fieldName) {
        this.fieldName = fieldName;
    }

    public string getValue() {
        return value;
    }

    public void setValue(string value) {
        this.value = value;
    }
}
}